react/presentation.html

```diff
@@ -2,43 +2,4710 @@
+                } from "./lifecycle";
+                import {
+                  UseStateComponent,
+                  UseEffectComponent,
+                  UseCallbackComponent,
+                  UseContextComponent,
+                  UseMemoComponent,
+                  UseRefComponent
+                } from "./hooks";
+
+                const blockStyle = { "border-style": "dotted" };
+
+                export default function App() {
+                  return (
+                    &lt;div className="App">
+                      &lt;div style={blockStyle}>
+                        <h1>Functional and class componets</h1>
+                        &lt;ClassCounter name={"Class Counter"} />
+                        &lt;FunctionalCounter name={"Functional Counter"} />
+                      &lt;/div>
+                      <br /> <br />
+                      &lt;div style={blockStyle}>
+                        <h1>Lifecycle examples</h1>
+                        &lt;MountingComponent name={"FirstName"} />
+                        &lt;UpdatingComponent />
+                        &lt;UnmountingComponent />
+                      &lt;/div>
+                      &lt;div style={blockStyle}>
+                        <h1>Hooks</h1>
+                        &lt;UseStateComponent />
+                        &lt;UseEffectComponent />
+                        &lt;UseCallbackComponent />
+                        &lt;UseContextComponent />
+                        &lt;UseMemoComponent />
+                        &lt;UseRefComponent />
+                      &lt;/div>
+                    &lt;/div>
+                  );
+                }              
+					    </code>
+            </pre>
+            <aside class="notes">
+              And the last thing we need to do is add our new component to
+              App.js.
+            </aside>
+          </section>
+          <section class="video-slide">
+            <h4 class="slide-name">useRef example</h4>
+            <div>
+              <iframe data-src="assets/videos/use_ref_example.mp4"></iframe>
+            </div>
+            <aside class="notes">
+              As you can see our component works fine. <br /><br />
+              It successfully counts the number of renderers. <br /><br />
+              Renderers work only if the state changes, if the value of the ref
+              changes, then the rerender does not start.
+            </aside>
+          </section>
+        </section>
+
+        <!-- CUSTOM HOOK EXAMPLE -->
+        <section>
+          <section class="numbering-and-picture-slide">
+            <h4 class="slide-name">custom hook example</h4>
+            <div>
+              <img
+                src="assets/images/custom_hook_example.png"
+                alt="custom_hook_example"
+              />
+              <ol>
+                <li><strong>use-custom-hook-component.js</strong> file</li>
+              </ol>
+            </div>
+            <aside class="notes">
```

Comment:
how to open the text from aside tag? I can't see it on running presentation

Reply:
you should add ```?showNotes``` to the url